src/js/components/Button/__tests__/__snapshots__/Button-kind-test.js.snap

```diff
@@ -20,6 +20,7 @@ exports[`Button kind border on default button 1`] = `
   border-style: solid;
   border-width: 2px;
   border-color: green;
+  padding: 2px 20px;
```

Comment:
This change might be problematic for users of the HPE theme in terms of backward compatibility. The button padding for the kind Button will immediately increase if the users upgrade to a grommet version that includes this PR unless we release the theme with a complimentary fix at the same time.

Reply:
If I remember correctly it doesn't because line 17 which comes from the 'kind' part of the theme is overriding it (really the aspect of the original problem.)